tests/fixtures/files/ingestion_jwt_response.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
lets use just one json for the tokens, we will add another only when we want a different response.